src/components/notebook-utils.js

```diff
@@ -58,6 +67,9 @@ export const notebookNameInput = ({ inputProps, ...props }) => h(ValidatedInput,
   }
 })
 
+export const getAnalysisType = name => name.endsWith('.ipynb') ? 'python' : 'r'
+
+export const getAnalysisFileExtension = analysisType => Utils.switchCase(analysisType, ['r', () => 'rmd'], ['python', () => 'ipynb'])
```

Comment:
Another option would be to make an object for mapping:
```suggestion
const typeExtensionMapping = { r: 'rmd', python: 'ipynb' }
export const getAnalysisFileExtension = analysisType => typeExtensionMapping[analysisType]
```

Reply:
Hmm... it would be elegant to do this, and then `_.invert` the object for extension to type mapping. I think I'll keep it as switch cases though, since it will allow us to be more flexible with handling new types (namely, we can use the default case as we're implementing new types if applicable)